src/playbacks/flashls/flashls.js

```diff
@@ -412,12 +412,16 @@ export default class FlasHLS extends BaseFlashPlayback {
     return duration
   }
 
-  seek(time) {
+  seekPercentage(percentage) {
     var duration = this.el.getDuration()
-    if (time > 0) {
-      time = duration * time / 100
+    if (percentage > 0) {
+      var time = percentage * 60 / duration
```

Comment:
maybe we can:
* start it with zero, outside of  ifs `var time = 0` in order to avoid `seek(undefined)`
* or put a check `!!time && this.seek(time)`

I'm in favor of the last